compiler_tester/src/test/case/input/deploy_evm.rs

```diff
@@ -184,22 +184,46 @@ impl DeployEVM {
             Err(error) => {
                 match error {
```

Comment:
Can't this match be avoided by just passing the whole error to the function?

Reply:
The thing is EVMError::Database does not implement Display, so in order to pass it to summary I need to match it.

Reply:
OK, let's keep it for now.
Thanks!